prov/efa/src/rdm/efa_rdm_pke_nonreq.c

```diff
@@ -643,6 +662,42 @@ void efa_rdm_pke_handle_eor_recv(struct efa_rdm_pke *pkt_entry)
 
 }
 
+/*
+ *   Sender handles the read NACK from receiver when receiver has failed to register the receive buffer
+ */
+void efa_rdm_pke_handle_read_nack_recv(struct efa_rdm_pke *pkt_entry)
+{
+	struct efa_rdm_read_nack_hdr *nack_hdr;
+	struct efa_rdm_ope *txe;
+	struct efa_rdm_peer *peer;
+
+	peer = efa_rdm_ep_get_peer(pkt_entry->ep, pkt_entry->addr);
+	assert(peer);
+	peer->num_read_msg_in_flight -= 1;
+
+	nack_hdr = (struct efa_rdm_read_nack_hdr *) pkt_entry->wiredata;
+
+	/* pre-post buf used here, so can NOT track back to txe with x_entry */
```

Comment:
Not quite follow the comment here. the `pkt_entry` is from receiver, how can we use `pkt_entry->x_entry` as txe?

Reply:
I copied this from the EOR packet processing code. I think it means what you're saying. I removed the comment.